docs/source/cli_help/add.rst

```diff
@@ -0,0 +1,25 @@
+add
+-----------
+Usage:
+::
+    git machete add [-o\|--onto=<target-upstream-branch>] [-R\|--as-root] [-y\|--yes] [<branch>]
+
+Adds the provided <branch> (or the current branch, if none specified) to the definition file.
+If <branch> is provided but no local branch with the given name exists:
+    * if a remote branch of the same name exists in exactly one remote, then user is asked whether to check out this branch locally (as in `git checkout`),
+    * otherwise, user is asked whether it should be created as a new local branch.
+
+If the definition file is empty or `-R`/`--as-root` is provided, the branch will be added as a root of the tree of branch dependencies.
```

Comment:
Hmmm AFAICS backtick `\`` renders _cursive_ rather than `preformatted` text... can this be changed? does .rst support in-line (as opposed to block) preformatted text at all? 🤔

Reply:
Oh, syntax highlighting isn't needed in the inline snippets

Reply:
Looks that the double backtick is what I meant here (just plain unhighlighted inline preformatted text)